features/step-definitions/sqlup-mode.el-steps.el

```diff
@@ -51,3 +51,8 @@
         (cl-letf (((symbol-function 'set-process-sentinel) #'ignore)
                   ((symbol-function 'get-buffer-process) #'ignore))
           (sql-interactive-mode))))
+
+(When "I type a newline"
+      "Simulate typing a newline character"
+      (lambda ()
+        (execute-kbd-macro (string-to-vector "\n"))))
```

Comment:
So after writing this I realised that the interactive sql mode probably remaps RET, and yep if I copy the test to `sql-interactive.feature` it fails. I'll need to figure out a fix for that.